src/site/_filters/live-posts.js

```diff
@@ -6,13 +6,11 @@ const {env} = require("../_data/site");
  * @return {boolean} Whether or not the post should go live.
  */
 module.exports = function livePosts(post) {
-  // If we ever wanted to enable featured posts we could uncomment these lines.
-  // See https://github.com/GoogleChrome/web.dev/pull/1222#issuecomment-516621561
-  // for an explanation on why this feature is currently disabled.
-  // const now = new Date();
-  // if (post.date > now) {
-  //   post.data.draft = true;
-  // }
+  const today = new Date();
+  today.setUTCHours(12, 0, 0, 0);
```

Comment:
Is the point of this line that it matches 7am PST? What about DST and friends?

Reply:
yeah. i added it based on one of your prior suggestions. idk, what do you recommend we do here?

https://twitter.com/rob_dodson/status/1121928275410866177